### DISCUSSION TITLE
Github Action - Postgres Matrix

### DISCUSSION THREAD

**Discussion by tedt10e** [bensheldon/good_job#399] [Ideas]
Hi @bensheldon,

I notice there is only one pg version (+ ruby 2.7 with pg 10.8) in the matrix.

Should pg 13 and 14 be in the matrix?

**Comment by bensheldon** [comment]
Yes, newer PG versions should be matrixed. Thank you if you're able to add them 🙏

**Reply by tedt10e** [reply]
Alrighty, I'll submit a PR. Thanks.
